src/libs/EmojiUtils.js

```diff
@@ -201,7 +202,12 @@ function replaceEmojis(text, isSmallScreenWidth = false) {
     for (let i = 0; i < emojiData.length; i++) {
         const checkEmoji = emojisTrie.search(emojiData[i].slice(1, -1));
         if (checkEmoji && checkEmoji.metaData.code) {
-            let emojiReplacement = checkEmoji.metaData.code;
+            let emojiReplacement;
+            if (preferredSkinTone >= 0) {
```

Comment:
```suggestion
            if (checkEmoji.metaData.types[preferredSkinTone]) {
```
For some emojis preferredSkinTone may not be available, so I think condition needs to be update

Reply:
Good catch @Pujan92 Was about to suggest something similar. The whole `types` array may be `undefined` which will cause a crash. Will work on it.

Reply:
I think we can just use lodashget here to be safe

Reply:
Ah right. I have just suggested something similar. Both should do it. Thanks! @esh-g

Reply:
Nice catch, thanks! I made that function but totally forgot about it for a sec